entrypoint.sh

```diff
@@ -31,21 +32,60 @@ secret_not_found() {
   exit 1
 }
 
+WRANGLER_VERSION=2
+
+# If no Wrangler version is specified install v2.
+if [ -z "$INPUT_WRANGLERVERSION" ]; then
+  npm i -g wrangler
+
+# If Wrangler version starts with 2 then install wrangler v2
+elif [[ "$INPUT_WRANGLERVERSION" == 2* ]] || [[ "$INPUT_WRANGLERVERSION" == "beta" ]]; then
```

Comment:
I'd flip this around. So:

- install v1 if `INPUT_WRANGLERVERSION == 1*`
- else, use v2.

We might introduce tags other than `beta` e.g. `wasm` or `pages` as we prototype stuff.